app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1286,9 +1288,27 @@ public void onBackPressed() {
             getSupportFragmentManager().popBackStack(NARROW,
                     FragmentManager.POP_BACK_STACK_INCLUSIVE);
         } else {
-            super.onBackPressed();
+            if(atHome){
```

Comment:
Can you help me figure out why atHome is needed? I tried running without it and it worked. I might have missed some use case.